Update ansible-matrix.md

kyliel opened the pull request:
1. Ansible 2.5 is released. Add 2.5 into the sheet.
2. Remove azure_module playbook. Per feedback, two playbook roles make people confused. Let us introduce azure_preview_module first. 
3. Correct the availability for azure_rm_appgwroutetable_facts.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Jim-Parker commented:
@MGoedtel Matt, can you take a look at Aleksandar's contribution and let me know if we're good to merge? If so, please sign off. thanks.

MGoedtel commented:
I just released an update to the article that will be merged today, so I am not going to sign-off on this and I'll make the changes to the updated article this evening and submit a new PR.

MGoedtel commented:
Alexandair, thanks for taking the time to review and contribute. I had already identified a couple of those items earlier and because I'm preparing to make additional updates to the article, I politely declined your PR and I'll incorporate your changes.

alexandair commented:
OK.

kyliel committed:
Update ansible-matrix.md

1. Ansible 2.5 is released. Add 2.5 into the sheet.
2. Remove azure_module playbook. Per feedback, two playbook roles make people confused. Let us introduce azure_preview_module first. 
3. Correct the availability for azure_rm_appgwroutetable_facts.
